website/docs/cloud-docs/api-docs/organizations.mdx

```diff
@@ -892,6 +894,183 @@ curl \
+| -------------------- | -------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization to update the data retention policy for.      |
+
+| Status  | Response                  | Reason(s)                                                      |
+| ------- | ------------------------- | -------------------------------------------------------------- |
+| [204][] | No Content                | Successfully updated the organization data retention policy.   |
+| [404][] | [JSON API error object][] | Organization not found, or user unauthorized to perform action.|
+
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+| Key path          | Type   | Default | Description                    |
+| ----------------- | ------ | ------- | ------------------------------ |
+| `data.type`       | string |         | A [data retention policy type](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies) |
+| `data.attributes` | object |         | The attributes for the [chosen policy type](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies) |
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "deleteOlderThanNDays": 33
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/organizations/hashicorp/relationships/data-retention-policy
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "drp-Kh74zpKVGNWbpugQ",
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "delete-older-than-n-days": 33
+    },
+    "relationships": {
+      "target": {
+        "data": {
+          "id": "hashicorp",
+          "type": "organization"
+        }
+      }
+    }
+  }
+}
+```
+
+## Remove data retention policy
+
+<EnterpriseAlert>
+This endpoint is exclusive to Terraform Enterprise, and not available in Terraform Cloud.
+</EnterpriseAlert>
+
+`DELETE /organizations/:organization_name/relationships/data-retention-policy`
+
+| Parameter            | Description                                                                                                                                                                          |
+| ---------------------| -------------------------------------------------------------------------- |
+| `:organization_name` | The name of the organization to remove the data retention policy for. |
+
+This endpoint removes the data retention policy explicitly set on an organization.
+Organizations without a data retention policy will default to using the site-wide data retention policy when determining how long to retain backing data.
```

Comment:
Non-Blocking Suggestion: Would it be beneficial to link the site admin docs here for easy access?:
```suggestion
Organizations without a data retention policy will default to using the [site-wide data retention policy](https://developer.hashicorp.com/terraform/enterprise/application-administration/general#data-retention-policies) when determining how long to retain backing data.
```